chain/network/src/network_protocol/borsh_conv.rs

```diff
@@ -153,7 +153,10 @@ impl TryFrom<&net::PeerMessage> for mem::PeerMessage {
 impl From<&mem::PeerMessage> for net::PeerMessage {
     fn from(x: &mem::PeerMessage) -> Self {
         match x.clone() {
-            mem::PeerMessage::Handshake(h) => net::PeerMessage::Handshake((&h).into()),
+            mem::PeerMessage::Tier1Handshake(_) => {
+                panic!("Tier1Handshake is not supported in Borsh encoding")
```

Comment:
comment that we don't have to support it - as we're working on deprecating borsch for network.

Reply:
done